plugins/sigma.renderers.customShapes/sigma.renderers.customShapes.js

```diff
@@ -53,12 +59,14 @@
       context.closePath();
       context.clip();
 
-      // Draw the actual image
-      context.drawImage(image,
-          x+Math.sin(-3.142/4)*r*xratio,
-          y-Math.cos(-3.142/4)*r*yratio,
-          r*xratio*2*Math.sin(-3.142/4)*(-1),
-          r*yratio*2*Math.cos(-3.142/4));
+      if(image.status =="ok") {
```

Comment:
Missing space (you might want to use `===` rather than `==` to optimize the condition).